data/os/Ubuntu/18.04.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
This whole file is redundant since `package_command` matches the default and `configure_epel` is set via the Debian family.

Reply:
updated